src/components/Application/ApplicationContent.tsx

```diff
@@ -2,12 +2,24 @@ import React from "react";
 import { Content as CarbonContent } from "carbon-components-react/lib/components/UIShell";
 import { CLASSPREFIX as eccgui } from "../../configuration/constants";
 
-function ApplicationContent({ children, isApplicationSidebarExpanded = false, ...restProps}: any) {
+interface IApplicationContentProps extends React.HTMLAttributes<HTMLElement> {
+    isApplicationSidebarExpanded?: boolean;
+    isApplicationSidebarRail?: boolean;
+}
+
+function ApplicationContent({
+    children,
+    isApplicationSidebarExpanded = false,
+    isApplicationSidebarRail = false,
+    ...otherProps
```

Comment:
Imho we need either a naming scheme for `otherProps` or put the HTML attributes in a separate parameter like `elementAttributes`.
Since this affects a lot of components, we should discuss this in a larger round. (CMEM-3742)

Reply:
This usually forwards properties that are not defined by the component itself, e.g. attributes for the inside used component (here we may double the property defs) or HTML attributes (then they don't need to repeated).